test/mpi/test_alltoallv.cc

```diff
@@ -140,6 +156,20 @@ TestAlltoallv::TestAlltoallv(ucc_test_team_t &_team, TestCaseParams &params) :
         args.src.info_v.displacements = (ucc_aint_t*)sdispls;
         args.dst.info_v.displacements = (ucc_aint_t*)rdispls;
     }
+    if (is_onesided) {
+        if (TEST_FLAG_VSIZE_64BIT == displ_bits) {
+            uint64_t *ldisp = (uint64_t *)calloc(nprocs, sizeof(uint64_t));
```

Comment:
can we check for `calloc` and `MPI_Alltoall` errors? here and in the else branch

Reply:
Added the checks. Thanks